src/scripts/mainApp.js

```diff
@@ -75,6 +75,8 @@ function msToTime(duration) {
     return hours + ":" + minutes + ":" + seconds
 }
 
+// FIXME: why are you requiring ipc renderer each time even after you already stored a reference in "var ipc"?
+// FIXME: never assigned soundPlayer...
```

Comment:
Ah ok :D Are you ok with using some linting to keep the code base formatted? We might need to add some unit tests later on to the application ^_^